src/components/Medicine/MedicationAdministration/MedicineAdminForm.tsx

```diff
@@ -329,7 +329,7 @@ export const MedicineAdminForm: React.FC<MedicineAdminFormProps> = ({
 
       <div className="space-y-2">
         <Label>{t("start_time")}</Label>
-        <div className="flex gap-2">
+        <div className="flex flex-row gap-2">
```

Comment:
isn't it flex-row by default? 🤔 correct me if i'm wrong

```suggestion
        <div className="flex gap-2">
```